test/completion/pep0484_typing.py

```diff
@@ -509,11 +509,33 @@ def dynamic_annotation(x: int):
 class Foo(typing.TypedDict):
     foo: str
     bar: typing.List[float]
+    an_int: int
     foo
     #! ['foo: str']
     foo
     #? str()
     foo
+    #? int()
+    an_int
+
+def typed_dict_test_foo(arg: Foo):
+    a_string = arg['foo']
+    a_list_of_floats = arg['bar']
+    an_int = arg['an_int']
+
+    #? str()
+    a_string
+    #? [float()]
```

Comment:
This line is wrong syntax, you should write `#? float()` instead of `#? [float()]`. For multiple results you could just write `#? float() str()`. It's separated by spaces.

Reply:
By `[float()]`, I mean a "list of floats". I don't know how many floats will be present in the result. If `[float()]` is the wrong syntax, how would I express an expected "list of floats"?

Reply:
It's not really possible to test this AFAIK with just one test. We could obviously make the test algorithm a bit smarter. However I usually test stuff like this with:

```
#? list()
a_list_of_strings
#? float()
a_list_of_strings[undefined_index]
```

BTW: I think you mean `a_list_of_floats`, so the name is wrong :)